src/components/layout/nav.js

```diff
@@ -277,9 +277,10 @@ const SignupButton = styled(Button)`
     opacity: 0;
 `
 
-const LinkSignupButton = styled(LinkButton)`
```

Comment:
It can't be extended from a button as it has the `to` and `external` prop behaving like a localizedlink.